usmqe_tests/api/gluster/test_cluster.py

```diff
@@ -74,27 +75,49 @@ def test_cluster_import_valid(valid_session_credentials):
 
         """
     nodes = api.get_nodes()
-    trusted_pool = storage.get_hosts_from_trusted_pool(nodes["nodes"][0]["fqdn"])
+    node_ids = None
+    for cluster in nodes["clusters"]:
+        if cluster["sds_name"] == "gluster":
+            node_ids = cluster["node_ids"]
+            break
+    node_fqdns = []
+    msg = "`sds_pkg_name` of node {} should be `gluster`, it is {}"
+    for node in nodes["nodes"]:
+        if node["node_id"] in node_ids:
+            pytest.check(node["detectedcluster"]["sds_pkg_name"] == "gluster",
+                         msg.format(node["fqdn"], node["detectedcluster"]["sds_pkg_name"]))
+            node_fqdns.append(node["fqdn"])
+    trusted_pool = storage.get_hosts_from_trusted_pool(node_fqdns[0])
     node_ids = [x["node_id"] for x in nodes["nodes"] if x["fqdn"] in trusted_pool]
     pytest.check(
         len(trusted_pool) == len(node_ids),
         "number of nodes in trusted pool ({}) should correspond \
         with number of imported nodes ({})".format(len(trusted_pool), len(node_ids)))
 
-    job_id = api.import_gluster_cluster(node_ids)["job_id"]
+    job_id = api.import_cluster(node_ids, "gluster")["job_id"]
 
     api.wait_for_job_status(job_id)
 
     integration_id = api.get_job_attribute(
         job_id=job_id,
-        attribute="integration_id",
+        attribute="TendrlContext.integration_id",
         section="parameters")
     LOGGER.debug("integration_id: %s" % integration_id)
 
+    # TODO(fbalak) remove this sleep after https://github.com/Tendrl/api/issues/159 is resolved.
+    import time
+    time.sleep(30)
+
+    imported_cluster = [x for x in api.get_cluster_list() if x["integration_id"] == integration_id]
```

Comment:
Why is this variable called `imported_cluster`, when it seems that it contains list of clusters which matches given id?